client/src/test/proto/spine/test/events/events.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package spine.test.events;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.spine.io";
+option java_package = "io.spine.test.client";
+option java_outer_classname = "EventsProto";
+option java_multiple_files = true;
+
+message ClCreateProject {
```

Comment:
It's misleading that a file called `events.proto` has one command and one value type in it.